kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/DbAuthzGroupService.java

```diff
@@ -63,7 +63,7 @@
 	/** Our log (commons). */
 	private static Logger M_log = LoggerFactory.getLogger(DbAuthzGroupService.class);
 	/** All the event functions we know exist on the db. */
-	protected Collection m_functionCache = new HashSet();
+	protected Collection<Object> m_functionCache = new HashSet<>();
```

Comment:
I think `m_functionCache` can be limited to just `String` as it should only contain function names.